packages/block-library/src/image/index.php

```diff
@@ -167,15 +167,17 @@ function block_core_image_render_lightbox( $block_content, $block ) {
 
 	// Note: We want to store the `src` in the context so we
 	// can set it dynamically when the lightbox is opened.
+	$img_width  = 'none';
+	$img_height = 'none';
 	if ( isset( $block['attrs']['id'] ) ) {
 		$img_uploaded_src = wp_get_attachment_url( $block['attrs']['id'] );
 		$img_metadata     = wp_get_attachment_metadata( $block['attrs']['id'] );
-		$img_width        = $img_metadata['width'];
-		$img_height       = $img_metadata['height'];
+		if ( ! empty( $img_metadata ) && is_array( $img_metadata ) ) {
+			$img_width  = $img_metadata['width'];
+			$img_height = $img_metadata['height'];
+		}
```

Comment:
This could cause an error since `img_width` and `img_height` won't be set `$img_metadata` doesn't pass the conditions.

`Undefined variable $img_width in /var/www/html/wp-content/plugins/gutenberg/build/block-library/blocks/image.php on line 216 Warning: Undefined variable $img_height in /var/www/html/wp-content/plugins/gutenberg/build/block-library/blocks/image.php on line 217`

Maybe we could provide a default value like the below else statement.

```suggestion
		$img_width        = $img_metadata['width'] ?? 'none';
		$img_height       = $img_metadata['height'] ?? 'none';
```

Reply:
Hey @ramonjd, Thanks for the feedback. I already moved `$img_width  = 'none';` and `$img_height = 'none';` out side of `if` statement. So, It won't throw error. I already checked the condition you mentioned with updated code.

Let me know, If I still need to make changes.

Reply:
By the way, Liked your short and optimal solution. I have updated the changes.

Thanks @ramonjd

Reply:
Thank you for working on this!

Looks good to me,  but maybe @artemiomorales should run an eye past it just to be sure.